_rest_routes.inc.php

```diff
@@ -139,6 +139,8 @@
  *              "user/surgery.write": "Write surgeries the user has access to (api:oemr)",
  *              "user/vital.read": "Read vitals the user has access to (api:oemr)",
  *              "user/vital.write": "Write vitals the user has access to (api:oemr)",
+ *              "user/transaction.read": "Read transactions the user has access to (api:oemr) (api:oemr)",
+ *              "user/transaction.write": "Write transactions the user has access to (api:oemr) (api:oemr)",
```

Comment:
probably very minor but does the `(api:oemr)` have to be duplicated at the end of these 2 lines @GetMoore ?

nice job with the styling btw!

Reply:
@GetMoore let's get the extra (api:oemr) removed and then it looks good enough to me to bring in.

Reply:
Done, thanks for checking this out guys!

Reply:
Sounds good.  Once the tests all come back green we'll bring it in.  Thanks for the contribution!